### PR TITLE
Initial commit of Terraform deployment to automatically export ADX Assets to S3

### PR DESCRIPTION
**Pull request by gerrardcowburn**
Initial commit of Terraform deployment to automatically export AWS Data Exchange Assets to S3 using Lambda and EventBridge

*Issue #, if available:*
#41 

*Description of changes:*
* Modified root level README.md to reference new contribution
* Added 'tf-auto-export-to-s3' folder under 'subscribers/python' folder

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ Examples of interacting with the AWS Data Exchange API from the data subscriber
 
 * [download-entitled-assets](subscribers/python/download-entitled-assets): Download all assets to which you've subscribed.
 * [pandas-describe-csv](subscribers/python/pandas-describe-csv): Download a CSV asset by Arn, import it into a Pandas data frame, and `describe()` the result.
+* [auto-export-to-s3](subscribers/python/tf-auto-export-to-s3): Terraform based deployment to automatically export newly published revisions to S3 using a CloudWatch Rule and Lambda
```
Comment (dblock) [inline README.md:35]: Let's put a period at the end of this sentence to feed my OCD ;)

File: subscribers/python/tf-auto-export-to-s3/README.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Auto Export To S3 (Python)
+
+This sample shows how to set up an AWS Lambda function which will automatically export all newly published revisions for a given Data Set to S3. All infrastructure is setup using Terraform.
```
Comment (dblock) [inline subscribers/python/tf-auto-export-to-s3/README.md:3]: Data Set -> DataSet, and a few more places like this

**Review by dblock — CHANGES_REQUESTED** [review]
Awesome, thank you! 

I think we need to extract the Lambda code as source in this PR and have a way to "build" it. Maybe a `build.sh` that invokes `zip` will suffice?

File: README.md (hunk 1 above)
Comment (gerrardcowburn) [reply README.md]: Done :) Also added them to the bullets in the README.me in the sub folder

File: subscribers/python/tf-auto-export-to-s3/README.md (hunk 2 above)
Comment (gerrardcowburn) [reply subscribers/python/tf-auto-export-to-s3/README.md]: Done except the reference in the EventBridge Rule notification detail.

**Comment by dblock** [comment]
Merged, thank you.

**Pull request merged**